src/bin/uhyve.rs

```diff
@@ -55,6 +55,16 @@ struct Args {
 	#[cfg(target_os = "linux")]
 	gdb_port: Option<u16>,
 
+	/// Paths that the kernel should be able to view, read or write.
+	///
+	/// Files and directories are separated using commas.
+	/// Desired mount paths must be explicitly defined after a colon.
+	///
+	/// Example: --file_map host_directory:/root/guest_directory,file.txt:/root/my_file.txt
+	#[arg(value_delimiter = ',')]
+	#[clap(long, env = "HERMIT_FILE_MAP")]
+	file_map: Option<Vec<String>>,
```

Comment:
I think it would be a better interface, if we have the separate mounts as separate options. Like `uhvye -v a.txt:b.txt -v c.txt:/bla/blub.txt`